pyneuroml/swc/ExportNML.py

```diff
@@ -0,0 +1,501 @@
+from typing import Dict, List, Optional, Set
+
+from LoadSWC import SWCGraph, SWCNode
```

Comment:
use a relative import here: `from .LoadSWC ..`, that will tell Python to look for `LoadSWC` in the same folder as `ExportNML`.

https://docs.python.org/3/reference/import.html#package-relative-imports